Resources/ServerInfo/Guidebook/Engineering/SingularityEngine.xml

```diff
@@ -0,0 +1,108 @@
+﻿<Document>
+  # Singularity Engine
+  The Singularity Engine is a powerful generator that uses a contained singularity to produce energy for the station.
+
+  <Box>
+    <GuideEntityEmbed Entity="SingularityGenerator"/>
+    <GuideEntityEmbed Entity="Singularity"/>
+  </Box>
+
+  It generates power by collecting the radiation emitted by the singularity using Radiation Collectors and gaseous plasma.
+
+  ## Starting the Singularity Engine
+  To start the Singularity Engine, engineers must follow these steps:
+  - Place the Singularity Generator in the middle of the containment field. Some stations start with the generator already in place, while others require engineers to bring it from cold storage.
+  - Turn on the four Containment Field Generators.
+  - Turn on and lock the Emitters.
+  - Confirm that the Containment Field Generators have created a Containment Field around the Singularity.
+  - Turn on the Radiation Collectors.
```

Comment:
Forgot to add that the collectors need plasma gas canisters in this step. Probably more important to note it first than later down in the same section.